code/src/components/IntroQuestion.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export const IntroQuestion = ({ nextQuestion }) => {
+	return (
+		<main className="main-container">
+			<div className="hidden-element"></div>
```

Comment:
Yes. Since I used space between for the vertical layout I needed an element in the top for the first question that doesn't have a back button. (For the rest of the question they all have 3 elements (back button in the top, form in the middle and next-button in the bottom). I gave it a class name of "hidden-element" just so I can remember it's purpose. (The flex is added to the parent).